cdc/scheduler/internal/v3/coordinator.go

```diff
@@ -256,7 +257,7 @@ func (c *coordinator) Close(ctx context.Context) {
 
 func (c *coordinator) poll(
 	ctx context.Context, checkpointTs model.Ts, currentTables []model.TableID,
-	aliveCaptures map[model.CaptureID]*model.CaptureInfo,
+	aliveCaptures map[model.CaptureID]*model.CaptureInfo, barrier *schedulepb.Barrier,
```

Comment:
How many tables in the barrier? Only for tables that have DDL or all tables?

Reply:
Only for tables that have DDL.

Reply:
Is there any limit? Eg. we have 10k tables that have DDL, how many tables will be included in a message?

Reply:
There is no limit right now, if you think it should be concerned,  I can add a limit for it.